templates/roles/gpu/tasks/main.yml

```diff
@@ -8,31 +8,13 @@
   apt_repository:
     repo: "deb http://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64 /"
 
-- name: Install CUDA 11.4
+- name: Install CUDA 11.5
+  - include_vars: versions.yml
   apt:
     name:
-      - cuda=11.4.0-1
-      - cuda-11-4=11.4.0-1
-      - cuda-runtime-11-4=11.4.0-1
-      - cuda-drivers=470.57.02-1
-      - cuda-drivers-470=470.57.02-1
-      - nvidia-driver-470=470.57.02-0ubuntu1
-      - libnvidia-gl-470=470.57.02-0ubuntu1
-      - nvidia-dkms-470=470.57.02-0ubuntu1
-      - nvidia-kernel-source-470=470.57.02-0ubuntu1
-      - nvidia-kernel-common-470=470.57.02-0ubuntu1
-      - libnvidia-common-470=470.57.02-0ubuntu1
-      - libnvidia-compute-470=470.57.02-0ubuntu1
-      - libnvidia-extra-470=470.57.02-0ubuntu1
-      - nvidia-compute-utils-470=470.57.02-0ubuntu1
-      - libnvidia-decode-470=470.57.02-0ubuntu1
-      - libnvidia-encode-470=470.57.02-0ubuntu1
-      - xserver-xorg-video-nvidia-470=470.57.02-0ubuntu1
-      - nvidia-modprobe=470.57.02-0ubuntu1
-      - nvidia-utils-470=470.57.02-0ubuntu1
-      - libnvidia-cfg1-470=470.57.02-0ubuntu1
-      - libnvidia-ifr1-470=470.57.02-0ubuntu1
-      - libnvidia-fbc1-470=470.57.02-0ubuntu1
+      - "cuda-libraries-11-5={{ cuda_libraries }}"
+      - "cuda-drivers-495={{ cuda_drivers }}"
+      - "cuda-compiler-11-5={{ cuda_compiler }}"
```

Comment:
I was thinking about something like `cuda-libraries-{{ cuda_libraries_version }}`, so if we need to update to `11.6`, we just need to update this variable

Reply:
Ok